modules/doc/content/newsletter/2023/2023_01.md

```diff
@@ -23,6 +23,10 @@ any discontinuous AD function. Normal finite volume variables do not support
 automatic porosity jump detection as they interpolate face values from cell
 center values.
 
+### Improved the ability to mark the converged solution invalid
```

Comment:
I appreciate the newsletter comment! One request though - could you place this in the February file (`2023_02.md`) since this didn't make it in during January?

Reply:
This is done in the new commit